### ISSUE TITLE
Change Download folder

### ISSUE THREAD

**Issue by RichieFlyUs** [kubilus1/autoortho#144]
Hi, I installed this and used the Setup options to specify scenery and cache directories on my E: drive. However when I commenced installation of an ortho set, the downloader began filling up my C: drive. I looked at the .autoortho config file in the root of my user profile and noted the download folder is specified but there is no program-based option to change this. I tried changing it in the config file, saving then closing and re-running the AutoOrth Setup, but it ignores the config file or reverts it and goes back to using the download folder at C:\Users\YOUR_USER\.autoortho-data\downloads.

There seems to be no way round this, and due to the quantity of data being downloaded it would be helpful to have access to this setting to define the download location successfully!

Thanks

Rich

**Comment by Simssmith** [comment]
Agree with this issue.  Is it possible to use a symlink for the folder :\Users\YOUR_USER.autoortho-data?

**Comment by kubilus1** [comment]
Yeah you could try linking to another dir, I imagine that would work.  Next release will allow this to be configurable, but that's still in the works.

**Comment by mckleashey123** [comment]
> Agree with this issue. Is it possible to use a symlink for the folder :\Users\YOUR_USER.autoortho-data?

I had the same issue. I made a symlink with with the (I ran this command into command prompt: mklink /D Symlink "C:\Users\MYUSER\.autoortho-data") whole folder including the sub folders and it didn't change a thing. Previously, I tried to just cut and paste the relevent folders/files to the drive that I wanted it to be in, instead of my C: drive. But instead it kept writing the config file to my C: drive. With the cache directory as well, I had the same sort of problem you were having.
